app/health.py

```diff
@@ -66,10 +66,10 @@ def determine_ftp_status(self):
             if conn:
                 logger.info("FTP health ok")
                 self.ftp_status = True
-        except FTPException as e:
-            logger.error("FTP exception raised", error=str(e))
-        except Exception as e:
-            logger.error("Unknown exception occurred when receiving ftp health", error=str(e))
+        except FTPException:
+            logger.exception("FTP exception raised")
+        except Exception:  # pylint: disable=broad-except
```

Comment:
Are we likely to receive other exceptions?

Reply:
I doubt it... but I don't know and don't want to pull something out without testing it out.